articles/faq/recipient-address-was-suppressed-due-to-customer-policy.md

```diff
@@ -24,8 +24,10 @@ You can use the API key to view who was on your account's Suppression List, but
 
 You can edit that by visiting this URL and clicking the Edit button for that particular API key and adding the Suppression Lists Read/Write permission: [https://app.sparkpost.com/#/account/credentials](https://app.sparkpost.com/#/account/credentials)
 
+Please see this artcile on how to manage your suppression list:
```

Comment:
spelling: `article`
We can just link to the article in our sentence instead of including the full URL as text. Try this:

`Please see [this article on how to manage your suppression list](https://www.sparkpost.com/docs/user-guide/using-suppression-lists/).`